src/main/Config.h

```diff
@@ -279,6 +279,11 @@ class Config : public std::enable_shared_from_this<Config>
     // index.
     size_t EXPERIMENTAL_BUCKETLIST_DB_INDEX_CUTOFF;
 
+    // When set to true, BucketListDB indexes are persisted on-disk so that the
+    // BucketList does not need to be reindexed on startup. Defaults to true.
+    // This should only be set to false for testing purposes
+    bool EXPERIMENTAL_BUCKETLIST_DB_PERSIST_INDEX;
```

Comment:
I was thinking this would be useful in supercluster, where we're already IO bound and don't want to do extra unnecessary disk IO when we'll never actually read the indexes from disk

Reply:
(Marking resolved as Marta is on vacation and I think this has been addressed)